src/components/calcite-link/calcite-link.e2e.ts

```diff
@@ -1,15 +1,48 @@
 import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
-import { accessible, renders } from "../../tests/commonTests";
+import { accessible, defaults, renders } from "../../tests/commonTests";
 
 describe("calcite-link", () => {
   it("renders", async () => renders("<calcite-link href='/'>link</calcite-link>", { display: "inline" }));
 
+  it("defaults", async () =>
+    defaults("calcite-link", [
+      {
+        propertyName: "download",
+        defaultValue: false
+      }
+    ]));
+
   it("is accessible", async () => {
     await accessible("<calcite-link href='/'>link</calcite-link>");
     await accessible("<calcite-link>link</calcite-link>");
     await accessible("<calcite-link icon-start='plus' icon-end='plus' href='/'>Go</calcite-link>");
   });
 
+  it("sets download attribute on internal anchor", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-link href="file.jpg">Continue</calcite-link>`);
+
+    const elementAsLink = await page.find("calcite-link >>> a");
+
+    expect(elementAsLink).not.toBeNull();
+    expect(elementAsLink).not.toHaveAttribute("download");
```

Comment:
For consistency, can these assertions use props vs attributes?

Reply:
Yes, but if I use the prop it will emit that console warning.

```
The state/prop "download" changed during rendering. This can potentially lead to infinite-loops and other bugs. 
```